### ISSUE TITLE
Historical system-wide perspective

### ISSUE THREAD

**Issue by patwater** [California-Data-Collaborative/cadc_reservoir_explorer#5]
A summary chart similar to the following chart from the Economist would be very helpful in contextualizing the reservoir supply picture.  One can imagine clicking on a reservoir in the map to see that band highlighted in this chart and vice versa:

![unnamed 1](https://user-images.githubusercontent.com/1587811/27191189-ea0626ee-51ab-11e7-9118-e03a721453db.png)

CC @christophertull @dmarulli @vr00n @datwater

**Comment by vr00n** [comment]
I can tableau towards this end  if the data is clean and if you think useful.

**Comment by patwater** [comment]
Great idea. Another vector would be to add a simple historical average dashed line to provide context on the Reservoir Explorer dashboard similar to this: https://cdec.water.ca.gov/cgi-progs/products/rescond.pdf

**Comment by patwater** [comment]
@mike-amodeo I agree with your idea that the historical average should be over the save time period as selected. So if November 5th is selected, the visualization would show the reservoir levels for that day in the year selected and then the historical average on November 5th. Similar logic for a date range say November 5th to December 5th.

CC @dmarulli

**Comment by mike-amodeo** [comment]
most recent commit has added the daily historical average to the line graph as a second line based on @dmarulli 's query. Some label placement still needs to be updated for clarity. Vertical bars on graph show each calendar year.
